src/frontend/htmlMockup.html

```diff
@@ -0,0 +1,72 @@
+<html><head><title> HANA Mockup </title>
+	<link rel="stylesheet" href="css/mockup.css">
+</head>
+<body>
+	<div id="firstLevelContainer">
+		<div id="appheader">
+			<div id="abbreviationMenue">
+				<span>BA</span>
+			</div>
+			<!-- TODO: search Feld muss Eingabefeld werden -->
+			<!-- TODO: "+ MenÃ¼" muss klickbar sein -->
+			<!-- TODO: "+ MenÃ¼" muss auf Klick aufklappen und Optionen zeigen -->
+			<!-- TODO: "+" Zeichen des MenÃ¼s muss angepasst oder durch Bild oder Symbol (Unicode Zeichen) ersetzt werden -->
+			<div id="menue">
+				✚
+			</div>
+			 <input name="search" id="searchField" placeholder="Search..." type="text">
+		</div>
+		<div id="appbody">
+			<div class="section">
+				<div class="sectionHeader">
+					Section 1
+				</div>
+				<div class="sourceEntity">
+					Clauss 2017a, Design Thinking: ...
+				</div>
+				<div class="sourceEntity">
+					Clauss 2017b, Comprehensive UX ...
+				</div>
+				<div class="sectionFooter">
+					...
+				</div>
+			</div>
+			<div class="section">
+				<div class="sectionHeader">
+					Section 2
+				</div>
+				<div class="sourceEntity">
+					London 2016, 
+				</div>
+				<div class="sourceEntity">
+					Etler 2015, The Art of Testing
+				</div>
+				<div class="sourceEntity">
+					Etler 2017, Dialog Systems: ...
+				</div>
+				<div class="sectionFooter">
+					...
+				</div>
+			<div class="section">
+				<div class="sectionHeader">
+					Section 3
+				</div>
+				<div class="sourceEntity">
+					Trost 2017, Management Approaches: ...
+				</div>
+				<div class="sourceEntity">
+					Zirpel 2017, Partner Ecosystems ...
+				</div>
+				<div class="sectionFooter">
+					...
+				</div>
+			</div>
+			</div>
```

Comment:
Ich glaube, dass das hier vor Zeile 50 soll.... oder soll eine `#section` in einer `#section` sein?